### ISSUE TITLE
Problem with a #hashtag after a character like " or ( or / or others

### ISSUE THREAD

**Issue by Neustradamus** [mastodon/mastodon#30823]
### Steps to reproduce the problem

Dear @mastodon team,

There is a #hashtag problem when we publish a #hashtag after a character like for example:
- "
- (
- /
- others

No problem on other social protocols/networks.

Temporary solution is to add a space.

Example here:
- https://mastodon.social/@neustradamus/112673485964973456

```
Test, not all example problems:
- #hashtag
- "#hashtag"
- " #hashtag"
- (#hashtag/#hashtag)
- ( #hashtag/#hashtag)
- ( #hashtag /#hashtag)
- ( #hashtag / #hashtag)
- -#hashtag
- _#hashtag
- !#hashtag
- ?#hashtag
- @#hashtag
- ;#hashtag
- ,#hashtag
- .#hashtag
- [#hashtag
- &#hashtag
- ^#hashtag
...
```

Can you solve it?

Thanks in advance.

### Expected behaviour

No problem

### Actual behaviour

Problem

### Detailed description

You can see here in post example:
- https://mastodon.social/@neustradamus/112673485964973456

The problems in post are:
```
- (#hashtag/#hashtag)
- ( #hashtag/#hashtag)
- ( #hashtag /#hashtag)
```
Of course other characters can have the same problem.

### Mastodon instance

*

### Mastodon version

*

### Browser name and version

*

### Operating system

*

**Comment by Cassolotl** [comment]
Duplicate of *Hashtags directly after / don't become links #1910* - a bug report that is over seven (7) years old. 😬 

Please add your comment or 👍 over there, and close this issue. Thank you!

**Comment by Neustradamus** [comment]
@renchap, @trwnh: Can you re-add "bug" tag?
Another duplicates always have.

**Comment by trwnh** [comment]
@Neustradamus No, there is now a separate "Bug" type. The "bug" label has been deleted.

**Comment by Neustradamus** [comment]
@trwnh: I see this:
- https://github.com/mastodon/mastodon/issues?q=type:%22Bug%22

**Comment by Neustradamus** [comment]
@trwnh: Good thanks :)

**Comment by Neustradamus** [comment]
@fishcharlie: Please look my ticket which is 2 years old soon.

@Cassolotl: You can see that my ticket has a lot of details :)

**Comment by Cassolotl** [comment]
@Neustradamus:

> @Cassolotl: You can see that my ticket has a lot of details :)

If the details you note here aren't in the top post or comments of the older issue, it would probably help to add them to the comments. :) Thank you!
